banking/src/components/features/BankReconciliation/BankEntryModalContent.tsx

```
@@ -0,0 +1,812 @@
+                         */
+                        const computeExpression = (expression: string) => {
+
+                            const script = `
+                                const transaction_amount = ${selectedTransaction.unallocated_amount ?? 0}
+                                ${expression};
+                            `
+
+                            let value = 0;
+
+                            try {
+                                value = window.eval(script);
```

Biome (2.4.15)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)